.github/workflows/linux.yml

```diff
@@ -245,6 +245,7 @@ jobs:
           source ${INSTALL_DIR}/setupvars.sh
           python -m pip wheel -v --no-deps --wheel-dir ${BUILD_DIR} \
               --config-settings=override=cross.arch="manylinux_2_31_x86_64" \
+              --find-links ${INSTALL_DIR}/tools \
```

Comment:
is it supposed we will drop `source ${INSTALL_DIR}/setupvars.sh` once main PR with dev files in OpenVINO wheel is merged?

Reply:
yes, we will remove setupvars.sh calls in future